main.go

```diff
@@ -207,10 +208,17 @@ func main() {
 	mux.Handle("/mdm/connect", connectHandler)
 
 	if checkEmptyArgs(*flUrl, *flScepUrl) {
-		logger.Log("warn", "Enrollment endpoint /mdm/enroll will be disabled because you did not specify flags for the external URL or SCEP URL")
+		logger.Log("warn", "Enrollment endpoint /mdm/enroll will be disabled because you did not specify flags/environment vars for the external URL (--url MICROMDM_URL) or SCEP URL (--scep-url/MICROMDM_SCEP_URL)")
```

Comment:
This was implemented in such a way that you could enroll with SCEP or use the same client identity for every single device (like in a test scenario).